sticky_notes.py

```diff
@@ -43,6 +43,27 @@ def sender(x):
                 while len(notes) >= 1:
                     note = notes.pop()
                     self.delay_task(len(notes),
-                                    sender(format_note(nick, note)))
+                                    self.sender(format_note(nick, note), event))
                 self.notes[nick] = []
                 return format_note(nick, last_note)
+
+    @cmd
+    def list_stickynotes(self, event):
+        """
+        Lists users with pending sticky notes if no argument is given, sticky
+        notes for given user otherwise.
+        """
+        args = event.args
+        format_note = lambda x, y: "{0}. {1}".format(x, y)
+
+        if len(args) == 0:
+            return 'I have sticky notes for: ' + ', '.join(self.notes.keys())
```

Comment:
The proper Python way is to use `.format` but I do understand the rationale here (cc @pepol )